cloudify_cli/commands/secrets.py

```diff
@@ -110,13 +110,12 @@ def get(key, tenant_name, logger, client):
 
 @secrets.command(name='export', short_help="Export secrets to a file")
 @cfy.options.encryption_password
-@cfy.options.not_encrypted
-@cfy.options.filter_by_visibility
+@cfy.options.visibility_filter
 @cfy.options.tenant_name_for_list(required=False,
                                   resource_name_for_help='secret')
 @cfy.options.all_tenants
 @cfy.options.filter_by
-@cfy.options.file_path
+@cfy.options.output_path
 @cfy.options.common_options
 @cfy.assert_manager_active()
 @cfy.pass_client()
```

Comment:
Add a doc string: "Export secrets from the Manager to a file. " so it will show when running `cfy sec export -h`